PresentScreenings/Controls/ScreeningLabel.cs

```diff
@@ -31,29 +29,18 @@ public ScreeningLabel(CGRect frame, Screening screening, bool withDay = false) :
             ColorView.SetScreeningColor(_screening, this);
             NeedsDisplay = true;
         }
-		#endregion
 
-        #region Override Methods
-        public override void MouseDown(NSEvent theEvent)
-        {
-            RaiseScreeningInfoAsked();
-        }
-        #endregion
-
-        #region Private Methods
         void Initialize()
         {
+            // Initialize control features.
             WantsLayer = true;
             LayerContentsRedrawPolicy = NSViewLayerContentsRedrawPolicy.OnSetNeedsDisplay;
         }
         #endregion
 
-        #region Events
-        public EventHandler ScreeningInfoAsked;
-
-        internal void RaiseScreeningInfoAsked()
+        #region Override Methods
+        public override void MouseDown(NSEvent theEvent)
         {
-            ScreeningInfoAsked?.Invoke(this, EventArgs.Empty);
         }
```

Comment:
Please consider to add a comment why this methods is overwritten by a no-op.

Reply:
Done.
The same bug, that is fixed by this override, still exists in `clickable labels` mode.
Far beyond this scope to fix.

Reply:
OK I created a bug issue for it.
No hurry to fix it.